llvm/lib/Support/Unix/Signals.inc

```diff
@@ -370,7 +371,7 @@ void sys::CleanupOnSignal(uintptr_t Context) {
 }
 
 // The signal handler that runs.
-static void SignalHandler(int Sig) {
+static void SignalHandler(int Sig, siginfo_t *Info, void *Context) {
```

Comment:
You can leave out Context since it is unused in this function, just keep the param unnamed.